### ISSUE TITLE
[$40][MSFT-237] [Challenge Feed][NVD][Edge] Sort By list is not read-out properly by the screen-reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2894]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges (Don't Login)
2. Press the tab key and go to the Sort by drop-down (Important Step)
3. Press on Sort by drop down
4. Move up/down with the arrow key and listen

**Expected Result**
Should read-out the names on the list by the screen-readers

**Actual Result**
Sort By list is not read-out properly by the screen-reader

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Edge

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=false&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F73

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096017 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096017 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096017 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096017 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096746 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by r0hit-gupta** [comment]
@sushilshinde @codeMinter @veshu unable to reproduce the issue. The video attached with the issue is already showing the correct way of how a select box should be read out by a screen reader. Please review.

Device: Desktop/Laptop
Operating System: Windows
Browser: Edge
Screen Reader Used: NVDA

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30096746 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by drasticdpk** [comment]
@r0hit-gupta , i am able to replicate it on Edge browser, can you please  re-verify it. See in google chrome it is working as expected.
https://monosnap.com/file/M7qiOjJDJ7i6DxxYg8FHJPhvJMRk85

**Comment by lakshmiathreya** [comment]
As per consensus from stakeholders, we are focusing exclusively on Chrome at this time. Putting this Issue on hold since it work on Chrome. fyi - @nkumar-topcoder @sushilshinde @drasticdpk @tosha5252

**Comment by crazyk07** [comment]
Creating separate issue for Edge and closing this issue.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096746<br/><br/>```This is an automated message for crazyk via Topcoder X```
